packages/data-transformer/src/normalizers/lowercase.normalizer.ts

```diff
@@ -0,0 +1,22 @@
+import {DataNormalizer} from "../interfaces/data-normalizer.interface";
+import {DataNormalizerUniqueKey} from "../types/data-normalizer-unique-key.type";
+import {LowercaseNormalizerOptions} from "./lowercase-normalizer.options";
+import {NormalizerInvalidSourceTypeError} from "../errors/normalizer-invalid-source-type.error";
+
+export class LowercaseNormalizer implements DataNormalizer<string, LowercaseNormalizerOptions>{
```

Comment:
I think we need to add the tag for it to be picked up and injected in the datatransformer
```suggestion
@tag("DataNormalizerInterface")
export class LowercaseNormalizer implements DataNormalizer<string, LowercaseNormalizerOptions>{
```